airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendGeographiesHandler.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.handlers;
+
+import io.airbyte.api.model.generated.Geography;
+import io.airbyte.api.model.generated.WebBackendGeographiesListResult;
+import java.util.Arrays;
+import java.util.Collections;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@AllArgsConstructor
+@Slf4j
+public class WebBackendGeographiesHandler {
+
+  public WebBackendGeographiesListResult listGeographiesOSS() {
+    // for now, OSS only supports AUTO. This can evolve to account for complex OSS use cases, but for
+    // now we expect OSS deployments to use a single default Task Queue for scheduling syncs in a vast
+    // majority of cases.
+    return new WebBackendGeographiesListResult().geographies(
+        Collections.singletonList(Geography.AUTO));
+  }
+
+  /**
+   * Only called by the wrapped Cloud API to enable multi-cloud
+   */
+  public WebBackendGeographiesListResult listGeographiesCloud() {
```

Comment:
remind me: why/how does the Cloud api call this?

Reply:
Is the idea to explicitly call this method in the wrapped server in Cloud?

Reply:
Yeah exactly, in the Cloud wrapped server, we'd call this specific endpoint instead of the OSS-specific endpoint. I think this is a short-term solution that we'll iterate on once we have more concrete needs around specialized OSS use-cases. I think it should serve our needs for now and should be pretty easy to adjust in the future